.github/workflows/ci.yml

```diff
@@ -388,14 +388,21 @@ jobs:
           path: |
             ${{ github.workspace }}
           retention-days: 7
-          
+  
+  build-services:
```

Comment:
If you want to test `make service`. You need refactor this job, a simple example:
```
  build-services:
    name: Build mo-service
    runs-on: ubuntu-latest
    steps:
      - name: checkout
        uses: actions/checkout@v2
        with:
          fetch-depth: '3'

      - name: Set up Go
        uses: ./.github/actions/setup_env

      - name: Build mo-service
        run: |
          cd $GITHUB_WORKSPACE && make service 
```

Reply:
Thanks. Done.